### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by rkristelijn** [jamrizzi/ignore-warnings#2]
First of all, thanks for looking into `npm WARN` and proposing a fix.

I'm investigating how to suppress certain npm warnings that I know I don't need the dependencies of, e.g. bootstrap. I just want to use the css and ignore jQuery and popper. 

```
npm WARN babel-core@7.0.0-bridge.0 requires a peer of @babel/core@^7.0.0-0 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.3.1 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yours
```

However I'm stuck at this step: https://github.com/codejamninja/ignore-warnings#usage 

What file do I need to create?

Also installing this package adds one warning:
```
npm WARN bootstrap@4.3.1 requires a peer of popper.js@^1.14.7 but none is installed. You must install peer dependencies yourself
```
Next to that I'm thinking about just installing jquery and popper, that also removes the warning. I don't have to use the code in node_modules, it just sits there, adds some space but removes the warning. What is your view on this approach? 

So TL;DR;
- [ ] please update the readme.md: add filename in https://github.com/codejamninja/ignore-warnings#usage
- [ ] please update readme.md: add motivation or differences with just installing the peer dependencies; e.g. webpack includes the peer dependencies in the packer, so this lib removes the warning and keeps the codebase small
